usuarios/delete-usuarios.spec.js

```diff
@@ -0,0 +1,98 @@
+const faker = require('faker')
+const testServer = require('../utils/testServer')
+const { postUser } = require('../utils/user')
+const { getAdminUserById } = require('../utils/user')
```

Comment:
você pode fazer o import na mesma linha
```suggestion
const { postUser, getAdminUserById } = require('../utils/user')
```